### ISSUE TITLE
When using dark theme Permission menu doesn't have text

### ISSUE THREAD

**Issue by poglazoff** [bitwarden/clients#5607]
### Steps To Reproduce

1. Use the Dark theme as the system theme. Set "Use system theme" in Bitwarden web vault settings.
2. Create New collection (but it works for existing collections also).
3. Go to the Access tab.
4. Add any user.
5. Try to change his Permission.
6. Find an almost empty dropdown list. The text appears when you move the mouse.

### Expected Result

See all text in the dropdown list.

### Actual Result

Can't see the text in the dropdown list.

### Screenshots or Videos

![image](https://github.com/bitwarden/clients/assets/1129843/45a807b1-f1b5-4350-a860-b9875058668e)
![image](https://github.com/bitwarden/clients/assets/1129843/9e674232-5e2e-480f-be98-3279d507cbc6)

### Operating System

Linux

### Operating System Version

Ubuntu 22.04.2 LTS

### Web Browser

Chrome

### Browser Version

Version 114.0.5735.106 (Official Build) (64-bit)

### Build Version

Version 2023.5.0

### Issue Tracking Info

- [ ] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Adedamola-Aina** [comment]
Hello @poglazoff 

We appreciate your bringing this to our attention. 

I have attempted to reproduce this issue on my Linux Ubuntu, Windows, and MacOS operating systems, but I don't appear to be experiencing it. Could it be that you are using a third-party extension for dark mode in your browser (e.g., dark reader and the likes)? Please disable this extension and try again using a browser without extensions. 

![image](https://github.com/bitwarden/clients/assets/91448810/006a6d39-9b9f-4164-b542-efb28a00a08d)

We use GitHub issues as a place to track bugs and other development related issues. If your issue persists, please write us back using our contact form, so we can continue troubleshooting: https://bitwarden.com/contact/ 

You can include a link to this issue in the message content.

**Comment by poglazoff** [comment]
After your message I tried another browser (Chromium Version 114.0.5735.106 (Official Build) snap (64-bit)) with no extensions, and even turned on Incognito mode. But the problem is still here.
First of all, I switched Bitwarden theme from "Use system theme" to "Dark" to exclude the possible influence of this option:
![image](https://github.com/bitwarden/clients/assets/1129843/5afc699f-1581-400d-8058-53d625577a91)
Then I reproduced the problem:
![image](https://github.com/bitwarden/clients/assets/1129843/22dc5c2d-4103-4bff-9fd0-65811857d7f4)
Maybe the problem is related to the use of Organization vault instead of Individual? I am using Organization vault.
I can provide Anydesk access to my desktop if you want.
